packages/nhsuk-frontend/src/nhsuk/components/card/macro-options.mjs

```diff
@@ -104,6 +104,11 @@ export const params = {
     required: false,
     description: 'Classes to add to the card.'
   },
+  id: {
+    type: 'string',
+    required: false,
+    description: 'Classes to add to the card.'
```

Comment:
Mind adjusting this text for ID?

```suggestion
    description: 'The ID of the card.'
```

<br>

We should make it the first param too, for consistency when it's rendered to HTML

It's what we do for character count, text input, select and texture